### PR TITLE
Added Docs for Android App & Labeled the iOS tutorial-0

### PR DESCRIPTION
**Pull request by Ry10p**
Sir, 

Please let me know if I have done anything wrong.

Thank you for your time,
-Ryan Carl

### CODE REVIEW

File: docs/tutorials/tutorial-0-Android.rst [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    $ cookiecutter https://github.com/pybee/briefcase-template
+
+This will ask a bunch of questions of you. We'll use an `app_name` of "tutorial_0", and a 
+`formal_name` of "Tutorial 0". Set the other values as you like.You'll now have a few files in this folder, including ``tutorial_0``. 
+
+Check out what the provided ``tutorial_0/app.py`` file contains:
+
+.. code-block:: bash
+
+    (BeeWare)user$ cd tutorial_0
```
Comment (glasnt) [inline docs/tutorials/tutorial-0-Android.rst]: Not everyone's setup will look like this. It's much easier for people to follow along if you just preface command line instructions with `$` only. 

E.g.: just `$ cd tutorial_0`

File: docs/tutorials/tutorial-0-Android.rst [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+emulated Android phone) and make sure you are still in the virtual environment
+we created at the start.  If you are then run:
+
+
+.. code-block:: bash
+
+  (Beeware)$ ./gradlew run
+
+
+....Poof.... your android apk is in the
```
Comment (glasnt) [inline docs/tutorials/tutorial-0-Android.rst]: As fun as we'd like tutorials to be, this language is overloaded. Please replace with "Tada!" or something similar.

File: docs/tutorials/tutorial-0-Android.rst [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+/android/build/outputs/apk/android-release-unsigned.apk.  You will
+still need to sign the apk, but you should see the phone running your android
+application!
+   
+And that is all, you created your first Android python app!
+Beeware is amazing right?
```
Comment (glasnt) [inline docs/tutorials/tutorial-0-Android.rst]: Again, please remove this line.

File: docs/tutorials/tutorial-0-Android.rst [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+    ./gradlew build
+
+Now its time to see what we have created.  Plug in your android phone(or
+emulated Android phone) and make sure you are still in the virtual environment
+we created at the start.  If you are then run:
+
+
+.. code-block:: bash
+
+  (Beeware)$ ./gradlew run
```
Comment (glasnt) [inline docs/tutorials/tutorial-0-Android.rst]: Remove `(Beeware)`, as above.

File: docs/tutorials/tutorial-0-Android.rst [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+This tutorial assumes you've read and followed the instructions in
+:doc:`/intro/getting-started`. This Tutorial is for Android Developers and Users!
+
+* A ``tutorial`` directory,
+* An activated Python 3 virtual environment
+* Briefcase installed in that virtual environment
+* We need gradle installed https://gradle.org/
+* The Android SDK and installation guid can be found at https://developer.android.com/studio/index.html
+* Ensure that the path to Android sdk is found.
+* THATS IT YOU ARE READY TO GO!
```
Comment (glasnt) [inline docs/tutorials/tutorial-0-Android.rst]: This line isn't required.

**Comment by Ry10p** [comment]
@glasnt 


Thanks for getting back to me **so fast** ill make the changes right now!


Thank you for your time,
-Ryan Carl

**Comment by freakboy3742** [comment]
@Ry10p A quick matter of protocol - Politeness is always appreciated, but there's no need to start every comment on Github with "Dear Sir", and end with "Yours, ...". If you want to draw a particular person's attention, starting with an `@someone` is enough (as I've done in this message to get your attention); the signoff is assumed because the message has come from you.

**Comment by Ry10p** [comment]
@freakboy3742  
Thank you so much for letting me know!

**Pull request merged**